server/errorHandler.ts

```diff
@@ -4,7 +4,10 @@ import logger from '../logger'
 
 export default function createErrorHandler(production: boolean) {
   return (error: HTTPError, req: Request, res: Response, next: NextFunction): void => {
-    logger.error(`Error handling request for '${req.originalUrl}', user '${res.locals.user?.name}'`, error)
+    logger.error(
+      `Error handling request for '${req.originalUrl}', user '${res.locals.user?.name}'`,
+      production ? null : error,
```

Comment:
I suspect this is the culprit but can't 100% be sure.

Reply:
Agree.